plgconfig.js

```diff
@@ -5,9 +5,9 @@ var config = {
   // folder.
   plgPath:        './build',
   plgCategory:    'MEI Export',
-  pluginFilename: 'sibmei2.plg',
+  pluginFilename: 'sibmei4.plg',
   linkLibraries: [
-    'libmei.plg', 'sibmei_batch_mxml.plg', 'sibmei_batch_sib.plg', 'sibmei_test_runner.plg'
```

Comment:
I don’t think libmei should be renamed here. Is there a reason for it?

Reply:
I renamed libmei to libmei4 so that sibmei4 can be installed side by side with sibmei2.  Like this, the user can generate both MEI 3 and MEI 4 if working with multiple MEI versions for different projects.  I'd suggest to make this approach standard for future major versions as well.

Reply:
As libmei is specific to an MEI version, there would be conflicts if both Sibmei versions tried to access the same libmei.

Reply:
OK... I do need to point out that the libmei plugin has been [generated from libmei itself](https://github.com/DDMAL/libmei/blob/master/tools/langs/manuscript.py). I haven't had time to update it in a while, but some attention should be paid to backwards compatibility, or coming up with an alternate method of updating `libmei.plg` from the schema when new MEI releases appear.

Reply:
@annplaksin updated libmei.plg to MEI 4 using libmei in 562e355.  I've also created some pull requests to libmei:

* https://github.com/DDMAL/libmei/pull/111
* https://github.com/DDMAL/libmei/pull/112
* https://github.com/DDMAL/libmei/pull/116

Can you explain your concerns regarding backwards compatibility?

Reply:
No problem!